.eslintrc

```diff
@@ -0,0 +1,87 @@
+{
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+        "modules": true
+    }
+  },
+  "extends": "eslint:recommended",
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "rules": {
+    "semi": [
+      2,
+      "always"
+    ],
+    "no-bitwise": 2,
+    "curly": 2,
+    "eqeqeq": 2,
+    "no-eq-null": 2,
+    "wrap-iife": [
+      2,
+      "inside"
+    ],
+    "no-use-before-define": [
+      2,
+      {
+        "functions": false
+      }
+    ],
+    "new-cap": 2,
+    "no-caller": 2,
+    "no-console": 0,
+    "no-new": 2,
+    "quotes": [
+      2,
+      "single",
+      {
+        "avoidEscape": true
+      }
+    ],
+    "space-before-function-paren": [
+      2,
+      "never"
+    ],
+    "comma-dangle": [
+      2,
+      "never"
+    ],
+    "eol-last": 2,
+    "keyword-spacing": [
+      2,
+      {}
+    ],
+    "space-before-blocks": [
+      2,
+      "always"
+    ],
+    "consistent-this": [
+      2,
+      "self"
+    ],
+    "indent": [
+      2,
+      4,
```

Comment:
Someday when it won't cause massive merge conflicts I'd be happy to move entirely to 2-space indents 😄